test/services/ReviewServiceSpec.scala

```diff
@@ -187,6 +195,10 @@ class ReviewServiceSpec extends UnitSpec with MockFactory with ReviewData with A
             .getById("validId")
             .returns(Future.successful(Left(Errors(NotFoundError))))
 
+          MockApprovalProcessReviewRepository
```

Comment:
If retrieving the process from approvalProcesses fails will this method actually get invoked?

Reply:
That does look strange - I'll check it out but thought the test would fail with unexpected call.

Reply:
I added a .never() to that call above and test still passed, I thought the calls defaulted to must be called once, but looks like the default is zero (or some other magic going on!) - will keep in mind going forward